composer.json

```diff
@@ -15,7 +15,8 @@
         }
     },
     "require": {
-        "php": ">=5.3.0"
+        "php": ">=5.3.0",
+        "predis/predis": "~1.0.1"
```

Comment:
The adapter should work withoutPredis too. You should move this dependency to "require-dev" and to "suggest". If possible you could add Phpredis to the suggest section to.